.template.config/template.json

```diff
@@ -1,9 +1,13 @@
 {
   "$schema": "http://json.schemastore.org/template",
   "author": "stacks@amido.com",
-  "classifications": [ "Stacks", "WebApi", "api" ],
+  "classifications": [ 
+    "Stacks", 
+    "WebApi" 
+  ],
   "name": "Amido Stacks Web Api - Full solution",
```

Comment:
Do we need the `- Full solution` as part of the name?

Reply:
Changed the template name to "Amido Stacks Web Api" as this now reflects the contents e.g. no infrastructure.